distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1204,6 +1204,98 @@ public void testNoEnforceValidityErrorsOrWarnings() throws Exception {
     }
 
     @Test
+    public void testQueryByErrorFailedValidators() throws Exception {
+        getServiceManager().startFeature(true,
+                "catalog-plugin-metacard-validation",
+                "sample-validator");
```

Comment:
@harrison-tarr removing this part caused these tests to fail. Looking over, the sample validator should be configured to be used at the start of the test. I saw other tests (that I admittedly wrote) stop it at the end of their run, so I reasoned that running one of these tests alone should pass. It didn't, however, so I added it back in.